Yosemite/Yosemite/Model/Mocks/MockStoresManager.swift

```diff
@@ -184,3 +208,32 @@ public class MockStoresManager: StoresManager {
         sessionManager.defaultSite = site
     }
 }
+
+private extension MockStoresManager {
+    func synchronizeGeneralSiteSettings(siteID: Int64) async {
+        await withCheckedContinuation { (continuation: CheckedContinuation<Void, Never>) in
```

Comment:
I had to explicitly declare the type of continuation, otherwise, the compiler will complain that it cannot infer the type. Please feel free to suggest another solution if possible, I'm still learning to use async-await so I could be wrong somewhere.

Reply:
yea I think this is fine, it's probably because there's no return value for the async function so the compiler can't determine the return type.